### PR TITLE
ci: ignore draft pull requests

### PR DESCRIPTION
**Pull request by redeboer**
Only run GitHub Actions if PR is ready for review (not draft status).

### CODE REVIEW

**Comment by redeboer** [comment]
Ok, works as expected:
- When in draft status, only PR linting and RTD is booted.
- Once marked ready for review, the rest of the checks are booted immediately, *without having to re-push*

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -81,8 +82,6 @@ jobs:
         run: make html
       - name: Test doctests in docstrings
         working-directory: doc
-        env:
-          NBSPHINX_EXECUTE: YES
```
Comment (spflueger) [inline .github/workflows/ci.yml:85]: why can this be removed now?

Comment (redeboer) [reply .github/workflows/ci.yml:85]: The doctest job should only check of `doctest` directives in the docstrings. Now it also runs the notebooks, which makes the workflow longer by some two minutes.

Note that the notebooks are already run in the job prior to it:
https://github.com/ComPWA/expertsystem/pull/289/files/7d18e93af3011db9576dc56a5a388b2e59e86400#diff-e9f950f17198d3d5e3122a44230a09b9R78-R82

Comment (spflueger) [reply .github/workflows/ci.yml:85]: i see

**Review by spflueger — APPROVED** [review]

**Pull request merged**